File: Lista03_HerancaEPolimorfismo/Heranca/app/src/main/java/Lista03/ProdutoNaoPerecivel.java
package Lista03;

public class ProdutoNaoPerecivel extends Produto{
    int tempoGarantia;

    public ProdutoNaoPerecivel(int codigo, double preco_unitario, String descricao, int quantidade_estoque, int tempoGarantia) {
        super(codigo, preco_unitario, descricao, quantidade_estoque);
        this.tempoGarantia = tempoGarantia;
    }

    public int getTempoGarantia() {
        return tempoGarantia;
    }
}

File: Lista11_Python/Ex04_Heranca/produto_nao_perecivel.py
from produto import *

class ProdutoNaoPerecivel(Produto):

    def __init__(self, codigo, preco_unitario, descricao, qtd_estoque, dataGarantia):
        super().__init__(codigo, preco_unitario, descricao, qtd_estoque)
        self._dataGarantia = dataGarantia

    def __str__(self):
        return super().__str__()

    def getGarantia(self):
        return self._dataGarantia

